strapi/strapi
# fix(content-releases): handle release.createdBy is null

### What does it do?

- Handles release.createdBy is null
- Renames the label of the 3 dots button from "Release actions" to "Release edit and delete menu". 
### Why is it needed?

- If the user who created the release is deleted then the page breaks

### How to test it?

- Create a user and give them acceess to releases
- Create a release with your new user
- Delete the user
- Go to the release they created
- Open the the edit and delete release menu (3 dots button).
- See the "Created by deleted user" message

### Related issue(s)/PR(s)

CS-601
